raspberryPI3/ros2_ws/src/car_control/src/car_control_node.cpp

```diff
@@ -148,15 +157,20 @@ class car_control : public rclcpp::Node {
 
             //Autonomous Mode
             } else if (mode==1){
```

Comment:
I think that later or soon we will have to make a function for manual mode and a function for autonomous mode, otherwise the autonomous branch will be very big.